modules/sdk-core/src/bitgo/enterprise/enterprise.ts

```diff
@@ -57,12 +57,18 @@ export class Enterprise implements IEnterprise {
    * Get the wallets associated with this Enterprise
    * @param params
    */
-  async coinWallets(params: Record<string, never> = {}): Promise<Wallet[]> {
-    const walletData = (await this.bitgo.get(this.baseCoin.url('/wallet/enterprise/' + this.id)).result()) as any;
-    walletData.wallets = walletData.wallets.map((w) => {
+  async coinWallets(params: ListWalletOptions = {}): Promise<Wallet[]> {
+    if (params.skip && params.prevId) {
+      throw new Error('cannot specify both skip and prevId');
+    }
+
+    const walletData = (await this.bitgo
+      .get(this.baseCoin.url('/wallet/enterprise/' + this.id))
+      .query(params)
+      .result()) as any;
+    return walletData.wallets.map((w) => {
```

Comment:
To avoid changing the function return type and cause a breaking change. I changed the return to just a list of wallets.